beacon-chain/core/validators/validator.go

```diff
@@ -78,9 +78,9 @@ func GetShardAndCommitteesForSlot(shardCommittees []*pb.ShardAndCommitteeArray,
 
 // AreAttesterBitfieldsValid validates that the length of the attester bitfield matches the attester indices
 // defined in the Crystallized State.
-func AreAttesterBitfieldsValid(attestation *pb.AggregatedAttestation, attesterIndices []uint32) bool {
+func AreAttesterBitfieldsValid(participationBitfield []byte, attesterIndices []uint32) bool {
```

Comment:
It's best to pass the actual item a function needs rather than a full struct when it only accesses one or two pieces of a struct, so changed it appropriately